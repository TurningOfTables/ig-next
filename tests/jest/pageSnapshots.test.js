import { render } from "@testing-library/react";
import Home from "../../pages/index";
import Showrunner from "../../pages/showrunner";
import ThisMerchantLife from "../../pages/this-merchant-life";
import Presskit from "../../pages/presskit";

it("renders Home page unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});

it("renders Showrunner page unchanged", () => {
  const { container } = render(<Showrunner />);
  expect(container).toMatchSnapshot();
});

it("renders This Merchant Life page unchanged", () => {
  const { container } = render(<ThisMerchantLife />);
  expect(container).toMatchSnapshot();
});

it("renders Presskit page unchanged", () => {
  const { container } = render(<Presskit />);
  expect(container).toMatchSnapshot();
});
