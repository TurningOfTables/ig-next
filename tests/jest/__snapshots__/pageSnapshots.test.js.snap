// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container mx-auto mt-10 text-center subpixel-antialiased"
  >
    <main>
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
      >
        <span
          style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
            style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
          />
        </span>
        <img
          alt="Inexplicable Games Logo"
          data-nimg="intrinsic"
          data-testid="header-image"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
      <p
        class="my-10 text-2xl"
        data-testid="intro-text-primary"
      >
        A one man games company based in the UK, working with talented freelancers from around the world.
      </p>
      <p
        class="max-w-md mx-auto text-xl italic"
        data-testid="intro-text-secondary"
      >
        <q>
          Two roads diverged in a wood, and I ... I took the one just covered with brambles, Got covered in cuts and barely made it out alive.
        </q>
         
        ~ Robert Frost
      </p>
      <div
        class="flex flex-row flex-wrap max-w-3xl mx-auto my-10 justify-center"
      >
        <a
          href="/showrunner"
        >
          <div
            class="flex justify-center m-5 w-64 h-64 bg-cover hover:shadow-2xl transition ease-in-out hover:scale-105 duration-200 bg-[url('/showrunnerCard.png')]"
          />
        </a>
        <a
          href="/this-merchant-life"
        >
          <div
            class="flex justify-center m-5 w-64 h-64 bg-cover hover:shadow-2xl transition ease-in-out hover:scale-105 duration-200 bg-[url('/tmlCard.jpg')]"
          />
        </a>
      </div>
    </main>
    <footer
      class="text-left mt-2 bottom-0"
      data-testid="footer"
    >
      Â© Inexplicable Games |
      <a
        href="/presskit"
      >
         Presskit
      </a>
    </footer>
  </div>
</div>
`;
